test/TEST-04-FULL-SYSTEMD/test-init.sh

```diff
@@ -1,21 +1,7 @@
 #!/bin/sh
 : > /dev/watchdog
-
-export PATH=/usr/sbin:/usr/bin:/sbin:/bin
-command -v plymouth > /dev/null 2>&1 && plymouth --quit
 exec > /dev/console 2>&1
 
-systemctl --failed --no-legend --no-pager > /run/failed
-
-ismounted() {
-    findmnt "$1" > /dev/null 2>&1
-}
-
-if ! ismounted /usr; then
-    echo "**************************FAILED**************************"
-    echo "/usr not mounted!!"
-    cat /proc/mounts >> /run/failed
-    echo "**************************FAILED**************************"
-fi
+usr/bin/systemctl --failed --no-legend --no-pager > /run/failed
```

Comment:
I think this should be `/usr/bin/systemctl'.

Reply:
Good catch.